projects/mdurl/fuzz_mdurl.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import atheris
+import mdurl
+
+
+def TestOneInput(data):
+  fdp = atheris.FuzzedDataProvider(data)
+  mdurl.parse(fdp.ConsumeUnicodeNoSurrogates(sys.maxsize))
```

Comment:
Never heard of maxsize. I always did float('inf')

Reply:
Interesting! It has carried with me since atheris came out due to: https://github.com/google/atheris/blob/770866ac436ece9aa2e9562f2a3f06086e2d8373/example_fuzzers/json_fuzzer/ujson_fuzzer.py#L41